src/components/friendly-iframe.js

```diff
@@ -94,6 +94,6 @@ export class FriendlyIframe {
    * @return {boolean}
    */
   isConnected() {
```

Comment:
Add unit tests for this method?

Reply:
Hmm.. There's an opportunity to split that into more specific tests that are unit testing isolated behavior. As is, I don't think it's testing the behavior of isConnected specifically, ie that the method returns the isConnected value of its element.

Reply:
Here's a new test:
https://github.com/subscriptions-project/swg-js/pull/2501/commits/a030094db15743c43ca5dd2abe964c58617f563e